src/domains/ecommerce-common/03_cosmosdb.tf

```diff
@@ -143,6 +143,15 @@ locals {
       ]
       shard_key = "_id"
     },
+    {
+      name = "user-stats"
```

Comment:
retention on this data? To delete records of inactive users? has it been considered?

Reply:
as per latest clarifications set TTL to 1 year with https://github.com/pagopa/pagopa-infra/pull/2453/commits/90b37decb1b2469cafe084b7870ea83594f441aa